addons/crm/static/src/js/forecast/forecast_models.js

```diff
@@ -0,0 +1,163 @@
+/** @odoo-module alias=crm.ForecastModels **/
+import KanbanModel from 'web.KanbanModel';
+
+/**
+ * Would be great if we could merge this with the `web` kanban_model
```

Comment:
As it is already somehow explained in first file, no need to be sorry in each sub-file :) .